app/utils/schema-blocks.ts

```diff
@@ -0,0 +1,92 @@
+import { assert } from '@ember/debug';
+import { SchemaBlock } from 'ember-osf-web/models/schema-block';
+
+export class QuestionChunk {
+  labelBlock?: SchemaBlock;
+  inputBlock?: SchemaBlock;
+  optionBlocks?: SchemaBlock[];
+  chunkId?: string;
+  answerId?: string;
+}
+
+function isEmpty(input: string | undefined) {
+    if (input === undefined || input === null || input === '') {
+        return true;
+    }
+    return false;
+}
+
+export function getPages(blocks: SchemaBlock[]) {
+    const pageArray = blocks.reduce(
+        (pages, block) => {
+            // instantiate first page if the schema doesn't start with a page-heading
+            if (pages.length === 0 && block.blockType !== 'page-heading') {
+                const blankPage: SchemaBlock[] = [];
+                pages.push(blankPage);
+            }
+
+            const lastPage: Array<SchemaBlock | QuestionChunk> = pages.slice(-1)[0];
```

Comment:
Pretty sure the union type here is left over from an earlier iteration.
```suggestion
            const lastPage: Array<SchemaBlock> = pages.slice(-1)[0];
```